exts/cesium.omniverse/cesium/omniverse/usdUtils/usdUtils.py

```diff
@@ -103,7 +103,13 @@ def add_cartographic_polygon() -> str:
     cartographic_polygon_path = Sdf.Path("/CesiumCartographicPolygons").AppendPath(name).pathString
     cartographic_polygon_path = omni.usd.get_stage_next_free_path(stage, cartographic_polygon_path, False)
 
-    UsdGeom.BasisCurves.Define(stage, cartographic_polygon_path)
+    basis_curves_prim = UsdGeom.BasisCurves.Define(stage, cartographic_polygon_path).GetPrim()
+    basis_curves_prim.GetAttribute("type").Set("linear")
+    basis_curves_prim.GetAttribute("wrap").Set("periodic")
+    basis_curves_prim.GetAttribute("points").Set(
+        [(-1000, 0, -1000), (-1000, 0, 1000), (1000, 0, 1000), (1000, 0, -1000)]
+    )
```

Comment:
Could you use the `UsdGeom.BasisCurves` functions instead of `GetAttribute`? There should be `GetTypeAttr`, `GetWrapAttr`, `GetPointsAttr`.

Reply:
I actually had those, but `GetTypeAttr` threw an error that it's not a member of `USDGeom.BasisCurves` so I resorted to the string. Let me double check that now that the code is more stabilized.

Reply:
I probably tried to use `SetTypeAttr`, which doesn't exist. At any rate, is this everyone's current state of IntelliSense in the Python code? 

![image](https://github.com/CesiumGS/cesium-omniverse/assets/1040194/8713cac5-31a3-4322-9c34-25ed8df12e14)

Reply:
Yes, I think it's a problem with Omniverse's Python bindings